decidim-elections/app/commands/decidim/votings/admin/create_polling_officer.rb

```diff
@@ -58,8 +58,6 @@ def create_polling_officer(user)
               voting: voting
             )
           end
-
-          # TODO: send notitfication to user??
```

Comment:
Shouldn't we notify the user?

Reply:
I was hesitant about this one because at this point we only know that the user has been added as an Officer to a Voting, which is quite vague.

I think it might be more informative for the user to get a notification when they are added as President/Manager to their Polling Station - that way they'll also be able to report an error if they get an email stating that they are assigned to a Polling Station they were not expecting.

What do you think @leio10?

Reply:
Of course we can do both things, but maybe we're sending too many emails and confuse the officer?